.github/workflows/deploy.yaml

```diff
@@ -40,24 +64,31 @@ jobs:
         run: |
           docker push ${IMAGE}
 
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build
+    if: always() && (needs.build.result == 'success' || needs.build.result == 'skipped')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
       - name: Deploy til q2
+        if: github.event.inputs.environment == 'q2' || (github.event_name == 'push' && github.ref == 'refs/heads/master')
         uses: nais/deploy/actions/deploy@v1
         env:
           APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
           CLUSTER: dev-fss
           RESOURCE: nais.yaml
           VAR: namespace=q2,ingress=https://melosys-soknad-mottak-q2.dev.intern.nav.no
-
       - name: Deploy til prod-fss
-        if: github.ref == 'refs/heads/master'
+        if: github.event.inputs.environment == 'p'
```

Comment:
Tenker at vi ikke nødvendigvis ønsker bare manuell deploy til prod for alle prosjekter.